### DISCUSSION TITLE
Is it possible to include a time tracker?

### DISCUSSION THREAD

**Discussion by ferrlen** [alainm23/planify#963] [Ideas]
I've just learned about this fantastic project! Although it's not really a time tracker, the only thing I find missing for my own needs would be a way to track how much time I'm spending on each task.

Would it be possible to include a time tracker for projects/tasks? (Or does such a feature/addon already exist?) The implementation could be super easy, just having a counter and a button for start / pausing / ending / editing time. 

As a suggestion, it could either be present in a project as a button or be a general tracking button somewhere (for example, on the side menu) that after the clock is stopped could be assigned to a project.

I think something along these lines was suggested here https://github.com/alainm23/planner/discussions/724 , but it's still vague and I haven't seen any movement there.

Thank you for your consideration!

**Comment by alainm23** [comment]
This is in development, thank you very much for commenting

**Reply by 0xh0russ** [reply]
Another cool feature would be timeblocking. Very effective planning technique. Would be cool if you can drag stuff from today's todo list into a column with a rectangle for each hour of the day. You can stretch stuff to span multiple hours. I can do it myself but I have to get familiar with the code first. 

Could double as a time tracker if you can resize stuff after the fact.

**Comment by DeimosHall** [comment]
Hey, I'd like to know if this is something that will be implemented or has been discarded.
